src/resources/index.html

```diff
@@ -131,20 +131,25 @@ <h2>{{route.params.name}}</h2>
    <canvas id="chartBt"></canvas>
   </div>
   <div style="grid-column: 1/-1">
+   <div v-show="false">{{metaDataKeys=["XBranch", "XVersion"]}}</div>
+   <div v-show="false">{{allJobs=jobsQueued.concat(jobsRunning).concat(jobsRecent)}}</div>
+   <div v-show="false">{{metaDataKeys=uniteMetadataKeys(allJobs)}}</div>
    <table class="striped">
     <thead><tr>
      <th><a class="sort" :class="(sort.field=='result'?sort.order:'')" v-on:click="do_sort('result')">&nbsp;</a></th>
      <th>Run <a class="sort" :class="(sort.field=='number'?sort.order:'')" v-on:click="do_sort('number')">&nbsp;</a></th>
      <th class="text-center">Started <a class="sort" :class="(sort.field=='started'?sort.order:'')" v-on:click="do_sort('started')">&nbsp;</a></th>
      <th class="text-center">Duration <a class="sort" :class="(sort.field=='duration'?sort.order:'')" v-on:click="do_sort('duration')">&nbsp;</a></th>
      <th class="text-center vp-sm-hide">Reason <a class="sort" :class="(sort.field=='reason'?sort.order:'')" v-on:click="do_sort('reason')">&nbsp;</a></th>
+     <th class="text-center vp-sm-hide" v-for="(value, key) in metaDataKeys">{{key}}<a class="sort" :class="(sort.field=='reason'?sort.order:'')" v-on:click="do_sort('reason')">&nbsp;</a></th>
```

Comment:
Do not sort by `reason`. It should be pretty simple to add sorting by metadata fields, using something like `'meta_' + key` for `sort.field` and in the backend using `ORDER BY metadata -> $key`, but also happy just to not implement sorting for metadata